Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -273,12 +284,25 @@ public final class AddNewCardViewModel: AddNewCardViewModelType, AddNewCardViewM
   public let newCardAdded: Signal<GraphUserCreditCard.CreditCard, Never>
   public let paymentDetails: Signal<PaymentDetails, Never>
   public let paymentDetailsBecomeFirstResponder: Signal<Void, Never>
-  public var rememberThisCardToggleViewControllerContainerIsHidden: Signal<Bool, Never>
-  public var rememberThisCardToggleViewControllerIsOn: Signal<Bool, Never>
+  public let rememberThisCardToggleViewControllerContainerIsHidden: Signal<Bool, Never>
+  public let rememberThisCardToggleViewControllerIsOn: Signal<Bool, Never>
   public let saveButtonIsEnabled: Signal<Bool, Never>
   public let setStripePublishableKey: Signal<String, Never>
   public let zipcodeTextFieldBecomeFirstResponder: Signal<Void, Never>
 
   public var inputs: AddNewCardViewModelInputs { return self }
   public var outputs: AddNewCardViewModelOutputs { return self }
 }
+
+// MARK: - Functions
+
+private func createPaymentSourceInput(
```

Comment:
I've been putting these helper functions into their own files as `CreatePaymentSourceInput+Constructor` - that way they can be tested separately.

Reply:
Ah cool, thanks!